forms.py

```diff
@@ -0,0 +1,24 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, SubmitField, PasswordField
+from wtforms.validators import DataRequired, Email, Length, EqualTo
+
+class SignUpForm(FlaskForm):
```

Comment:
Out of curiosity did you test the validators? If so, how should I? I'm trying to make sure that we get converage in that spot, but feel like I"m missing something.